sdk/spring/scripts/get_spring_boot_managed_external_dependencies.py

```diff
@@ -2,11 +2,12 @@
 # This script is used to get all 3rd party dependencies managed by spring-boot-dependencies and spring-cloud-dependencies.
 #
 # How to use this script.
-#  1. Update `SPRING_BOOT_VERSION` and `SPRING_CLOUD_VERSION` in this script manually.
+#  1. Get `SPRING_BOOT_VERSION` and `SPRING_CLOUD_VERSION`.
```

Comment:
Add description about how to get the version, like sdk/spring/scripts/README.md.